_sass/home.scss

```diff
@@ -24,30 +25,34 @@
   }
 
   h3 {
-    font-size: 26px;
-  }
-  .features h3 {
-    height: 2em;
```

Comment:
Please restore 2em in order to align the value statements.
It might be worth adding a comment to explain why it is here.